### PR TITLE
Feature/29 setup extension

### PR DESCRIPTION
**Pull request by BeejLuig**
- styled popup.html
- manual conversion
- supports USD, EUR, CAD
- API calls pointing directly to localhost for now

### CODE REVIEW

File: src/extension/popup.html [hunk 1]
```diff
@@ -1,41 +1,125 @@
...
-            <option value="EUR">Euro</option>
-            <option value="USD">United States dollar</option>
-            <option value="CAD">Canadian dollar</option>
-        </select>
-    </label>
-    <pre id="data"></pre>
+    <section class="container">
+        <input id="fromAmount" name="amount" type="number" value="0">
+        <div class="select">
+            <select id="fromCode" name="from" id="fromCode">
```
Comment (kerrykimrusso) [inline src/extension/popup.html]: Duplicate `id` attr

File: src/extension/popup.html [hunk 2]
```diff
@@ -1,41 +1,125 @@
...
-        <input name="toAmount" type="number" id="toAmount" value="0.0">
-        <select name="toCode" id="toCode">
-            <option value="EUR">Euro</option>
-            <option value="USD">United States dollar</option>
-            <option value="CAD">Canadian dollar</option>
-        </select>
-    </label>
-    <pre id="data"></pre>
+    <section class="container">
+        <input id="fromAmount" name="amount" type="number" value="0">
```
Comment (kerrykimrusso) [inline src/extension/popup.html]: Is this going to be part of the extension or was it for testing purposes?

File: src/extension/popup.js [hunk 3]
```diff
@@ -5,17 +5,38 @@ const {
...
+    console.log('to:', toCode);
+    const conversionRate = await fetch(
+      `http://localhost:3000/api/v1/currencies/code/${fromCode}`
+    )
+      .then(res => res.json())
+      .then(data => data.currency.convertTo[toCode]);
+    console.log('conversion rate:', conversionRate);
+    fromAmountInput.value = formatAmount(fromAmount * conversionRate);
+  })
+);
```
Comment (kerrykimrusso) [inline src/extension/popup.js]: Similar to my question above, is this for testing purposes?

File: src/extension/popup.html (hunk 2 above)
Comment (BeejLuig) [reply src/extension/popup.html]: Yeah, this isn't part of the ultimate objective of the extension, but it's the easiest thing to implement and might be a nice-to-have feature down the road

**Review by kerrykimrusso — APPROVED** [review]

**Pull request closed without merging**